spec/fortnox/api/repositories/invoice_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+require 'fortnox/api/repositories/invoice'
+require 'fortnox/api/models/invoice'
+
+describe Fortnox::API::Repository::Invoice do
+
+  before{
+    ENV['FORTNOX_API_BASE_URL'] = ''
+    ENV['FORTNOX_API_CLIENT_SECRET'] = ''
+    ENV['FORTNOX_API_ACCESS_TOKEN'] = ''
+  }
+
+  after{
+    ENV['FORTNOX_API_BASE_URL'] = nil
+    ENV['FORTNOX_API_CLIENT_SECRET'] = nil
+    ENV['FORTNOX_API_ACCESS_TOKEN'] = nil
+  }
```

Comment:
These should probably be moved into a helper as well ... We'll need them for every repository spec.